CONTRIBUTING.rst

```diff
@@ -338,11 +379,15 @@ If you are adding new framework flavor support, you'll need to modify ``pytest``
 
 4. ``travis/large-requirements.txt``: add your framework and version to the list of requirements
```

Comment:
We shoul rename `traivs` folder because we no longer use it. I'll make a PR.

Reply:
Okay, I'll wait for your PR to make that change since I can't rename it until your PR is merged.

Reply:
Addressed by #3253